content/en/docs/tasks/administer-cluster/kubelet-in-userns.md

```diff
@@ -49,6 +49,21 @@ Rootless Podman is not supported.
 
 <!-- Supporting rootless podman is discussed in https://github.com/kubernetes/minikube/issues/8719 -->
 
+## Running Kubernetes inside Unprivileged Containers
+
+### sysbox
+
+[Sysbox](https://github.com/nestybox/sysbox) is an open-source container runtime
+(similar to "runc") that supports running system-level workloads such as Docker
+and Kubernetes inside containers isolated via the Linux user namespace.
+
+See Kubernetes-in-Docker in the [Sysbox docs][https://github.com/nestybox/sysbox/blob/master/docs/quickstart/kind.md] for more info.
+
+Note that Sysbox supports running Kubernetes inside unprivileged containers
+without requiring Cgroup v2 and without the `KubeletInUserNamespace` feature
+gate. It does this by exposing specially crafted proc and sys filesystems inside
+the container plus several other advanced techniques.
```

Comment:
Reviewers: take care to check whether this change fits the [content guide](https://kubernetes.io/docs/contribute/style/content-guide/)

Reply:
@ctalledo : Per the content guide linked by @sftim above, this is third party content and therefore, will require a third party content tag to be added. 

{{% thirdparty-content %}}